styles.scss

```diff
@@ -554,21 +565,28 @@ footer {
     width: 22rem;
     height: auto;
   }
-  .w-50 {
-    width: 99% !important;
+  .w-50.w-50 {
+    width: 99%;
   }
   .w-100 {
```

Comment:
You're still modifying bootstrap classes: w100 is supposed to give an element a width of 100% because that's its functionality and you should not alter that instead you should create your own class or target your element with another CSS selector  (non blocking comment) keep this in mind for your following projects